render.js

```diff
@@ -0,0 +1,34 @@
+const Renderer = function () {
+    const renderPosts = function (posts) {
+        $("#posts").empty()
+        for (let post of posts) {
+            const postContainer = createPostContainer(post)
+            $("#posts").append(postContainer)
+        }
+    }
+
+    return {
+        renderPosts
+    }
+}
+
+const createPostContainer = function (post) {
+    const postContainer = $(`<div class="post" data-id=${post.id}></div>`)
```

Comment:
You should place this method inside the module

Reply:
but this function will never be used outside the renderer, should i put it in the module any way ?
and should i return it  ? 
thanks for the feedback, really appreciate it